File: hack-2023/src/jsonObjects/Employer.ts
export class Employer {
    private email: string;
    private prefix: string;
    private fName: string;
    private lName: string;
    private suffix: string;
    private majors: string[];

    constructor(email: string, prefix: string, fName: string, lName: string, suffix: string, status: number, majors: string[]) {
        this.email = email;
        this.prefix = prefix;
        this.fName = fName;
        this.lName = lName;
        this.suffix = suffix;
        this.majors = majors;
    }
}

File: backend/src/Employer.java
public class Employer extends Person {
    /**
     * Creates a new {@code Employer}.
     * @param fName the first name of this {@code Employer}.
     * @param lName the last name of this {@code Employer}.
     * @param prefix the salutation for this {@code Employer}.
     * @param suffix the suffix for this {@code Employer}.
     * @param email the contact email for this {@code Employer}.
     * @param university the employed {@code University} for this {@code Employer}.
     */
    public Employer(String fName, String lName, String prefix, String suffix, String email,
                    University university) {
        super(fName, lName, prefix, suffix, email, university);
    }
}
